compiler/BUILD.bazel

```diff
@@ -1,12 +1,53 @@
+load("//:java_grpc_library.bzl", "java_rpc_toolchain")
+
```

Comment:
nit: I think you should set a default visibility of private here, and explicitly declare the vis on the rest of the targets.

Reply:
Private is the default, and why repeat it again to each of the targets?